packages/wdio-allure-reporter/src/index.js

```diff
@@ -249,9 +249,11 @@ class AllureReporter extends WDIOReporter {
         test.setDescription(description, type)
     }
 
-    addAttachment({ name, content, type = 'text/plain' }) {
-        if (!this.isAnyTestRunning()) {
-            return false
+    addAttachment({ name, content, type = 'text/plain' }, loadExistingAttachmentInAfterHook = false) {
+        if (!loadExistingAttachmentInAfterHook) {
```

Comment:
```suggestion
        if (!loadExistingAttachmentInAfterHook || !this.isAnyTestRunning()) {
```
no need for two nested ifs

Reply:
No, we need to ensure the nested condition is not interpreted at all if the flag is true...  it's logically incorrect to use `||` or `&&` in this case.